crates/proto/src/rr/rdata/caa.rs

```diff
@@ -471,7 +471,7 @@ pub fn read_issuer(bytes: &[u8]) -> ProtoResult<(Option<Name>, Vec<KeyValue>)> {
                         state = ParseNameKeyPairState::BeforeKey(key_values);
                     }
                     // push onto the existing key
-                    ch if !ch.is_control() && !ch.is_whitespace() => {
+                    ch if ('\x21'..='\x3A').contains(&ch) || ('\x3C'..='\x7E').contains(&ch) => {
```

Comment:
Since this looks quite a bit less obvious than the previous version, mind adding one or two lines of comments here to explain what this means in practice?